### PR TITLE
docs: update version/doctor output

### PR DESCRIPTION
**Pull request by efiop**
Per https://github.com/iterative/dvc/pull/4848

> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

File: content/docs/command-reference/version.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ usage: dvc version [-h] [-q | -v]
 | `Supports`                                  | Types of [remote storage](/doc/command-reference/remote/add#supported-storage-types) supported by the current DVC setup (their required dependencies are installed)  |
 | `Cache types`                               | [Types of links](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache) supported (between <abbr>workspace</abbr> and <abbr>cache</abbr>)     |
 | `Cache directory`                           | Filesystem type (e.g. ext4, FAT, etc.) and drive on which the <abbr>cache</abbr> directory is mounted                                                                |
+| `Caches`                                    | [Types of caches](/doc/user-guide/managing-external-data) (e.g. local, s3, ssh, etc.) configured in the repo                                                        |
+| `Remotes`                                    | [Types of remotes](/doc/command-reference/remote/add) (e.g. local, s3, ssh, etc.) configured in the repo                                                            |
```
Comment (efiop) [inline content/docs/command-reference/version.md:23]:
```suggestion
| `Remotes`                                    | [Types of remotes](/doc/command-reference/remote/add) (e.g. local, s3, ssh, etc.) configured in the repo                                                           |
```

File: content/docs/command-reference/version.md [hunk 2]
```diff
@@ -107,12 +109,14 @@ Inside a DVC project:
 ```dvc
 $ dvc version
 
-DVC version: 1.7.4 (pip)
+DVC version: 1.10.0 (pip)
```
Comment (efiop) [inline content/docs/command-reference/version.md:112]: just to not confuse people with the older DVC version

File: content/docs/command-reference/version.md [hunk 3]
```diff
@@ -107,12 +109,14 @@ Inside a DVC project:
...
 ---------------------------------
 Platform: Python 3.8.3 on Linux-5.4.0-47-generic-x86_64-with-debian-buster-sid
 Supports: gs, hdfs, http, https, s3
 Cache types: hardlink, symlink
 Cache directory: ext4 on /dev/sda1
+Caches: local
+Remotes: https
```
Comment (efiop) [inline content/docs/command-reference/version.md:119]:
```suggestion
Remotes: https, ssh
```

File: content/docs/command-reference/version.md [hunk 4]
```diff
@@ -19,6 +19,8 @@ usage: dvc version [-h] [-q | -v]
 | `Supports`                                  | Types of [remote storage](/doc/command-reference/remote/add#supported-storage-types) supported by the current DVC setup (their required dependencies are installed)  |
 | `Cache types`                               | [Types of links](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache) supported (between <abbr>workspace</abbr> and <abbr>cache</abbr>)     |
 | `Cache directory`                           | Filesystem type (e.g. ext4, FAT, etc.) and drive on which the <abbr>cache</abbr> directory is mounted                                                                |
+| `Caches`                                    | [Types of caches](/doc/user-guide/managing-external-data) (e.g. local, s3, ssh, etc.) configured in the repo                                                        |
+| `Remotes`                                    | [Types of remotes](/doc/command-reference/remote/add) (e.g. local, s3, ssh, etc.) configured in the repo                                                           |
```
Comment (efiop) [inline content/docs/command-reference/version.md:23]:
```suggestion
| `Remotes`                                   | [Types of remotes](/doc/command-reference/remote/add) (e.g. local, s3, ssh, etc.) configured in the repo                                                             |
```

**Comment by jorgeorpinel** [comment]
Finally looking at this! Sorry for the delay.

File: content/docs/command-reference/version.md (hunk 2 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/version.md:112]: No strong opinion but there's another 3 instances of `1.7` in this doc.

File: content/docs/command-reference/version.md [hunk 5]
```diff
@@ -19,6 +19,8 @@ usage: dvc version [-h] [-q | -v]
 | `Supports`                                  | Types of [remote storage](/doc/command-reference/remote/add#supported-storage-types) supported by the current DVC setup (their required dependencies are installed)  |
 | `Cache types`                               | [Types of links](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache) supported (between <abbr>workspace</abbr> and <abbr>cache</abbr>)     |
 | `Cache directory`                           | Filesystem type (e.g. ext4, FAT, etc.) and drive on which the <abbr>cache</abbr> directory is mounted                                                                |
+| `Caches`                                    | [Types of caches](/doc/user-guide/managing-external-data) (e.g. local, s3, ssh, etc.) configured in the repo                                                         |
```
Comment (jorgeorpinel) [inline content/docs/command-reference/version.md:22]:
```suggestion
| `Caches`                                    | Cache [location types](/doc/user-guide/managing-external-data) configured in the repo (e.g. local, SSH, S3, etc.)                                                         |
```
>Needs formatting.

File: content/docs/command-reference/version.md [hunk 6]
```diff
@@ -19,6 +19,8 @@ usage: dvc version [-h] [-q | -v]
...
+| `Remotes`                                   | [Types of remotes](/doc/command-reference/remote/add) (e.g. local, s3, ssh, etc.) configured in the repo                                                             |
```
Comment (jorgeorpinel) [inline content/docs/command-reference/version.md:23]:
```suggestion
| `Remotes`                                   | Remote [location types](/doc/command-reference/remote/add#supported-storage-types) configured in the repo (e.g. SSH, S3, Google Drive, etc.)                                                             |
```
> Also needs formatting.

**Review by jorgeorpinel — APPROVED** [review]
Left specific suggestions to commit + format.

Assuming those are addressed, I'm approving this, since it reflects core. But it makes things pretty confusing. E.g. Cache types (link types, closer to `cache.type` config.) vs. Caches (external cache types, closer to remote types). I opened iterative/dvc/issues/4990 for that.

**Review by jorgeorpinel — COMMENTED** [review]
OK I guess my review doesn't really qualify as approval haha. But almost

File: content/docs/command-reference/version.md (hunk 2 above)
Comment (efiop) [reply content/docs/command-reference/version.md:112]: Updated, thanks!

**Comment by shcheklein** [comment]
@jorgeorpinel any updates on this? ready to be merged?

**Comment by efiop** [comment]
Oops, forgot to ping after addressing the comments. :(

**Comment by jorgeorpinel** [comment]
Sorry got tangled up in other things, checkingggg

**Review by jorgeorpinel — APPROVED** [review]

**Pull request merged**